Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM openshift/origin-release:golang-1.10
-COPY . /go/src/github.com/openshift/cluster-image-registry-operator
-RUN cd /go/src/github.com/openshift/cluster-image-registry-operator && \
-    go build ./cmd/cluster-image-registry-operator
+FROM openshift/origin-release:golang-1.10 as builder
+WORKDIR /go/src/github.com/openshift/cluster-image-registry-operator
+COPY . .
+RUN make build
 
 FROM centos:7
-
 RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
-
-COPY --from=0 /go/src/github.com/openshift/cluster-image-registry-operator /usr/bin
-COPY deploy/image-references deploy/00-crd.yaml deploy/01-namespace.yaml deploy/03-operator.yaml deploy/02-rbac.yaml /manifests/
```

Comment:
why is this removed?  it's necessary for the installer to process our operator image correctly.

Reply:
Because these files aren't used by the operator. I added a note that these files are for the installer.